src/com/javafee/task/bankaccount/BankAccount.java

```diff
@@ -0,0 +1,32 @@
+package src.com.javafee.task.bankaccount;
+
+
+public class BankAccount {
+
+
+    private double balance;
+
+    public double getBalance() {
+        return balance;
+    }
+
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+    void withdraw(double withdraw_anmount) {
+        balance -= withdraw_anmount;
+
+    }
+
+    void withdraw(int percentage) {
+        double percent = Double.parseDouble(String.valueOf(percentage)) / 100.0;
+        double amount = balance * percent;
+        this.withdraw(amount);
```

Comment:
In that case and similar cases you can make it shorter like:
```
        this.withdraw(balance * (Double.parseDouble(String.valueOf(percentage)) / 100.0));
```
Try to use such constructions in other places if possible, please.